src/AccessibilityInsights.SharedUx/Telemetry/TelemetryProperty.cs

```diff
@@ -40,5 +40,8 @@ public enum TelemetryProperty
         InstalledDotNetFrameworkVersion,
         OsArchitecture,
         CustomUIAPropertyCount,
+        IsAutoDetectColorContrastEnabled,
```

Comment:
This seems really wordy, and since we filter first by `TelemetryAction`, it's nice if the property names are easily reusable. How about
```suggestion
        IsNowEnabled,
```